content/automate/ManageIQ/Container/Openshift/Operations/Methods.class/__methods__/available_projects.rb

```diff
@@ -0,0 +1,54 @@
+#
+# Description: provide the dynamic list content from available flavors
+#
+module ManageIQ
+  module Automate
+    module Container
+      module Openshift
+        module Operations
+          class AvailablePorjects
+            def initialize(handle = $evm)
```

Comment:
@lfu Can we add a spec test for this method?
There are examples here:
https://github.com/ManageIQ/manageiq-content/tree/master/spec/content/automate/ManageIQ/Infrastructure/VM/Transform/Import.class/__methods__